contracts/contracts/BidderRegistry.sol

```diff
@@ -391,15 +388,13 @@ contract BidderRegistry is
         feePercent = newFeePercent;
     }
 
-    /**
-     * @dev Withdraw funds to the fee recipient.
+    /** 
+     * @notice Sets the new fee payout period in blocks
+     * @dev onlyOwner restriction
+     * @param newFeePayoutPeriodBlocks The new fee payout period in blocks
      */
-    function withdrawFeeRecipientAmount() external nonReentrant {
-        uint256 amount = feeRecipientAmount;
-        feeRecipientAmount = 0;
-        require(amount != 0, "fee amount is zero");
-        (bool successFee, ) = feeRecipient.call{value: amount}("");
-        require(successFee, "fee transfer failed");
+    function setNewFeePayoutPeriodBlocks(uint256 newFeePayoutPeriodBlocks) external onlyOwner {
+        protocolFeeTracker.payoutPeriodBlocks = newFeePayoutPeriodBlocks;
```

Comment:
It's a good practice to generate event in case that function is called
```suggestion
         protocolFeeTracker.payoutPeriodBlocks = newFeePayoutPeriodBlocks;
         emit FeePayoutPeriodBlocksUpdated(newFeePayoutPeriodBlocks);
```